app/views/admin/facility_groups/_form.html.erb

```diff
@@ -16,7 +16,10 @@
         </div>
       <% end %>
 
-      <%= form.select(:organization_id, organizations.map { |organization| [organization.name, organization.id] }) %>
+      <% unless organizations.empty? %>
```

Comment:
We're assuming that if you have FG admin permissions, you can edit your FG, but you can reassign the FG to a different Org. Based on this assumption, we'd removed the drop-down for Org select from FG edit if there are no Orgs that you can actually manage.

Reply:
Hehe neat optimization